src/client/Home.css

```diff
@@ -0,0 +1,49 @@
+.Home {
+  text-align: center;
+}
+
+.Home-logo {
+  animation: Home-logo-spin infinite 20s linear;
+  height: 80px;
+}
+
+.Home-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+@keyframes Home-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.Table {
+  border: solid;
+  margin-top: 36px;
+}
+
+.Table  th {
+  background-color: grey;
+}
+
+.Table td {
+  padding: 10px;
+}
+
+.Home-footer  {
+  height: 100px;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  position: fixed;
```

Comment:
The `position` is duplicate in this CSS rule.
Maybe you'd like to have only one.